src/store/persona/slice.ts

```diff
@@ -15,11 +21,22 @@ const personaSlice = createSlice({
   name: 'persona',
   initialState: PersonaState,
   reducers: {
-    cleanLogout: (_) => PersonaState,
+    cleanLogout: (state) => {
+      state.profile = undefined;
```

Comment:
I think that `null` is best suited. it is up to you.

Reply:
I think `null` should not be used and `undefined` should priorized https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined